src/main/java/gregtech/common/covers/filter/SimpleFluidFilter.java

```diff
@@ -53,13 +52,15 @@ public void configureFilterTanks(int amount) {
 
     @Override
     public @NotNull Widget<?> createWidgets(PanelSyncManager syncManager) {
-        return new Row().coverChildrenHeight().widthRel(1f)
+        return Flow.row().coverChildrenHeight().widthRel(1f)
                 .child(SlotGroupWidget.builder()
                         .matrix("FFF",
                                 "FFF",
                                 "FFF")
-                        .key('F', i -> new FluidSlot()
-                                .syncHandler(new FixedFluidSlotSH(filterReader.getFluidTank(i)).phantom(true)))
+                        .key('F', i -> new GTFluidSlot()
+                                .syncHandler(GTFluidSlot.sync(filterReader.getFluidTank(i))
+                                        .phantom(true)
+                                        .showAmount(false)))
```

Comment:
Could this `showAmount` here be messing with the Advanced Voiding Cover?

Reply:
somewhat, i'll need to figure out a way to pass information from the regulator/voiding cover down to the filter's sync handler